barcodes_generator_product/__manifest__.py

```diff
@@ -7,7 +7,7 @@
 {
     "name": "Generate Barcodes for Products",
     "summary": "Generate Barcodes for Products (Templates and Variants)",
-    "version": "14.0.1.1.0",
+    "version": "16.0.1.1.0",
```

Comment:
16.0.1.0.0

Reply:
Still pending

Reply:
Hi @pedrobaeza : don't know why github doesn't put a flag "obsolete" here but this diff is wrong.
The version is correct. see : https://github.com/OCA/stock-logistics-barcode/pull/450/files

this PR can be merged.

Reply:
Something should happened in between, as if you go to the mentioned URL, the version is not the correct one.

Reply:
?!? Well, when I go to that url (https://github.com/OCA/stock-logistics-barcode/pull/450/files) I see : 
```    "version": "16.0.1.1.0",```


Note : Ci is green and major version is checked by the CI AFAIK.

Reply:
It should be `16.0.1.0.0`, which is the original claim.

Reply:
ow sorry for the noice.
Fixed.